components/GothamistArticle.vue

```diff
@@ -218,8 +222,22 @@
 <script>
 import gtm from '@/mixins/gtm'
 import disqus from '@/mixins/disqus'
+import { differenceInMonths } from 'date-fns'
 import LazyHydrate from 'vue-lazy-hydration'
-import RelatedAuthors from './RelatedAuthors.vue'
+import ImageWithCaption from 'nypr-design-system-vue/src/components/ImageWithCaption.vue'
```

Comment:
you shouldn't have to import any components anymore - Nuxt should auto import everything. Was that not working?

Reply:
was getting tons of warnings in the console whenever i ran tests